_sass/sidebar.scss

```diff
@@ -0,0 +1,29 @@
+.sidebar-left {
+  ul {
+    font-size:90%;
+    li {
+      list-style-image: none;
+      margin: 0;
+      background: $COLOR_GRAY;
+      line-height: 150%;
```

Comment:
@borisyankov we can remove it, I copy-pasted this entirely from binary-static since I was instructed to make it look the same.